modules/gnetBidAdapter.js

```diff
@@ -1,9 +1,9 @@
-import { _each, parseSizesInput, isEmpty } from '../src/utils.js';
 import { registerBidder } from '../src/adapters/bidderFactory.js';
+import * as utils from '../src/utils.js';
```

Comment:
We standardized only importing the needed functions. Can you go back to the previous structure for utils?

Reply:
Alright, fixed. Appreciated!